extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/indexing/DurableStorageCleaner.java

```diff
@@ -75,66 +72,58 @@ public boolean isEnabled()
   }
 
   @Override
-  public void schedule(ScheduledExecutorService exec)
+  public void run()
   {
-    LOG.info("Starting the DurableStorageCleaner with the config [%s]", config);
+    try {
```

Comment:
Same comment for the other duties/helpers which are wrapping the entire duty in a try..catch block.

Reply:
Thanks for pointing this out! You are right, neither of the existing duty/helpers are doing anything special with the exception, just logging it or wrapping it up and rethrowing it. I will get rid of these.

Reply:
Edit: The implementation of both the existing duties actually throw unchecked IOException, and it is best to just handle them in the respective duties rather than adding `IOException` to the signature of `OverlordDuty.run()`.

Reply:
> we could just add a plain Exception to the method signature.

Went ahead with this. Thanks for the suggestion!